rfd/0109-cloud-agent-upgrades.md

```diff
@@ -327,6 +327,30 @@ to be changed to follow the same pattern of registering a release channel and in
 by ec2 discovery.
 
 
+### ECS Fargate Agents
+
+When a user uses the Teleport Discover flow to connect an RDS, a Teleport agent is automatically provisioned using ECS Fargate. We need a
```

Comment:
To avoid "user uses"
```suggestion
When Teleport Discover flow is used to connect an RDS, a Teleport agent is automatically provisioned using ECS Fargate. We need a
```